src/main/java/org/springframework/data/couchbase/core/query/Query.java

```diff
@@ -127,6 +128,26 @@ public Query with(final Pageable pageable) {
 		return with(pageable.getSort());
 	}
 
+	/**
+	 * queryScanConsistency
+	 *
+	 * @return queryScanConsistency
+	 */
+	public QueryScanConsistency getConsistency() {
```

Comment:
do we need a public getter on this?

Reply:
It's used by ExecutableFindByQuerySupport.matching(Query) - if Query has scanConsistency set, it is taken from there instead of requiring a separate call to ExecutableFindByQuerySupport.consistentWith( scanConsistency ).